CooperationHullMainSite/Views/Home/Index.cshtml

```diff
@@ -1,73 +1,108 @@
-﻿
-<!-- Hero -->
-<section class="hero" >
+﻿<!-- Hero -->
+<section class="hero">
 
-        <div class="row">
+    <div class="row">
 
-            <div class="hero-image-container">
+        <div class="hero-image-container">
 
             <div class="d-none d-lg-block">
-                    <img src="~/assets/img/blackandwhitehero.png" alt="" />
-                </div>
-                <div class="d-lg-none ">
-                    <img src="~/assets/img/pattern 1.jpeg" alt="" />
-                </div>
+                <img src="~/assets/img/blackandwhitehero.png" alt="" />
             </div>
+            <div class="d-lg-none ">
+                <img src="~/assets/img/pattern 1.jpeg" alt="" />
+            </div>
+        </div>
 
-                <div class="position-absolute">
-
-                    <div class="col-12 col-md-11 col-lg-9 col-xl-7 col-xxl-6 text-left hero-text">
-                        <h1 class="heading-xl">THE BIG QUESTION</h1>
-                        <p class="heading-sm">Strong, caring communities.</p>
-                        <p class="heading-sm">Enough food for everyone.</p>
-                        <p class="heading-sm">A healthy planet.</p>
-                        <p class="heading-sm">Lives filled with meaning</p>
-                        <br />
-                        <p class="heading-sm">This is the kind of society we all want.</p>
-                        <p class="heading-sm">So why don't we build it?</p>
+        <div class="position-absolute">
 
-                        <button type="button" class="btn btn-light btn-lg px-4 gap-3">Sign the pledge</button>
-                    </div>
-                </div>
+            <div class="col-12 col-md-11 col-lg-9 col-xl-7 col-xxl-6 text-left hero-text">
+                <h1 class="heading-xl">THE BIG QUESTION</h1>
+                <p class="heading-sm">Strong, caring communities.</p>
+                <p class="heading-sm">Enough food for everyone.</p>
+                <p class="heading-sm">A healthy planet.</p>
+                <p class="heading-sm">Lives filled with meaning</p>
+                <br />
+                <p class="heading-sm">This is the kind of society we all want.</p>
+                <p class="heading-sm">So why don't we build it?</p>
 
+                <button type="button" class="btn btn-light btn-lg px-4 gap-3">Sign the pledge</button>
+            </div>
         </div>
 
+    </div>
+
 </section>
 
 
 
-<section class="homepage-block big-answer ">
+<section class="homepage-block the-plan ">
     <div class="row">
 
-         <div class="col-lg-6 col-md-12 p-4 order-lg-1 order-md-0">
+        <div class="col-lg-6 col-md-12 p-4">
+            <h1 class="heading-lg">What's the plan?</h1>
```

Comment:
I think semantically these would be better of as `h2` elements. `h1` is usually reserved for the main heading on a page, and so its best to stick with that convention, especially for users who need assistive technologies that rely on hierarchy.